src/control/cmd/daos/container.go

```diff
@@ -1297,20 +1297,24 @@ func (cmd *containerSetPropCmd) Execute(args []string) error {
 	}
 	defer deallocCmdArgs()
 
-	ap.props = cmd.Args.Props.props
```

Comment:
I think that you'll need to add a `defer cmd.Args.Props.Cleanup()` here in order to make NLT happy.

Reply:
Good catch. I guess it was previously getting cleaned up as part of the `ap` struct.